logr.gemspec

```diff
@@ -1,17 +1,17 @@
 # coding: utf-8
 lib = File.expand_path('../lib', __FILE__)
 $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
-require 'structured_logger/version'
+require 'logr/version'
 
 Gem::Specification.new do |spec|
-  spec.name          = "structured_logger"
-  spec.version       = StructuredLogger::VERSION
+  spec.name          = "logr"
+  spec.version       = Logr::VERSION
   spec.authors       = ["Peter Marton"]
   spec.email         = ["martonpe@secretsaucepartners.com"]
 
-  spec.summary       = "Structured logging for events and metrics"
-  spec.description   = "Structured logging for events and metrics"
-  spec.homepage      = "https://github.com/sspinc/structured_logger"
+  spec.summary       = "Structured logging with events and metrics"
+  spec.description   = "Structured logging with events and metrics"
+  spec.homepage      = "https://github.com/sspinc/logr"
   spec.license       = "MIT"
```

Comment:
I'll check with our lawyer about the appropriate open source license before we push this to Rubygems.